pkg/controller/atlasdeployment/serverless_deployment.go

```diff
@@ -89,3 +90,13 @@ func isTagsEqual(a []*mongodbatlas.Tag, c []*mongodbatlas.Tag) bool {
 	}
 	return false
 }
+
+func updateServerlessRequest(tags *[]*mongodbatlas.Tag, serverlessSpec *mdbv1.ServerlessSpec) *mongodbatlas.ServerlessUpdateRequestParams {
```

Comment:
note: you can get rid of this method that only fills the struct and move the body to the place where it's called

Reply:
done